docker-compose.yml

```diff
@@ -7,12 +7,12 @@ services:
       context: .
     image: hmcts/ccd-definition-designer-api
     environment:
-      - DEFINITION_STORE_DB_HOST=ccd-definition-store-database
-      - DEFINITION_STORE_DB_PORT=5432
-      - DEFINITION_STORE_DB_USERNAME
-      - DEFINITION_STORE_DB_PASSWORD
-      - DEFINITION_STORE_IDAM_KEY
-      - DEFINITION_STORE_S2S_AUTHORISED_SERVICES=ccd_data,ccd_gw
+      - DEFINITION_DESIGNER_DB_HOST=ccd-definition-store-designer
```

Comment:
@adr1ancho - I believe this should be `ccd-definition-designer-database`, instead of `ccd-definition-store-designer`